core/vbl/vbl_local_minima.hxx

```diff
@@ -83,14 +83,30 @@ bool local_minima(vbl_array_2d<T> const& in, vbl_array_2d<T>& minima, T thresh)
         lm = in[r+1][c]   - in[r][c];
         lr = in[r+1][c+1] - in[r][c];
         dmin = mval;
-        if (ul<=thresh) continue; if (ul<dmin) dmin = ul;
-        if (um<=thresh) continue; if (um<dmin) dmin = um;
-        if (ur<=thresh) continue; if (ur<dmin) dmin = ur;
-        if (lf<=thresh) continue; if (lf<dmin) dmin = lf;
-        if (ri<=thresh) continue; if (ri<dmin) dmin = ri;
-        if (ll<=thresh) continue; if (ll<dmin) dmin = ll;
-        if (lm<=thresh) continue; if (lm<dmin) dmin = lm;
-        if (lr<=thresh) continue; if (lr<dmin) dmin = lr;
+        if (ul<=thresh) continue;
+	if (ul<dmin) { dmin = ul; }
```

Comment:
This indentation is surprising. Shouldn't it be flush with the rest of the code in this block?

Reply:
#@$!@#$! tabs.  It looks right in my editor, but the tab character was there.